CsvParser/csvParser.go

```diff
@@ -129,3 +132,32 @@ func makeJsonObjFile(arr []incident.Incident) *os.File {
 
 	return f
 }
+
+// // Helper functions to store incidents in according directory
+func addIncidentToJSON(inc incident.IncidentData, jsonFileName string) {
```

Comment:
Can you remind me why this function is here? Is @hmc-elephant-s18 taking advantage of this or is it duplicate code at this point?

Reply:
I'm not sure... @hmc-elephant-s18

Reply:
I don't know why It is there since I wrote my own version in a separate branch. I think this was probably written when Amy was trying to write code to add an incident to a .json file.